firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -208,12 +210,37 @@ final class HomepageViewController: UIViewController,
             }
             headerCell.applyTheme(theme: themeManager.getCurrentTheme(for: windowUUID))
             return headerCell
+        case .pocket:
+            switch item {
+            case .pocket(let story):
+                guard let pocketCell = collectionView?.dequeueReusableCell(
+                    withReuseIdentifier: PocketStandardCell.cellIdentifier,
+                    for: indexPath
+                ) as? PocketStandardCell
+                else {
+                    return UICollectionViewCell()
+                }
+                pocketCell.configure(item: story, theme: themeManager.getCurrentTheme(for: windowUUID))
+                return pocketCell
+            case .pocketDiscover(let title):
+                guard let pocketCell = collectionView?.dequeueReusableCell(
+                    withReuseIdentifier: PocketDiscoverCell.cellIdentifier,
+                    for: indexPath
+                ) as? PocketDiscoverCell
+                else {
+                    return UICollectionViewCell()
+                }
+                pocketCell.configure(text: title, theme: themeManager.getCurrentTheme(for: windowUUID))
+                return pocketCell
```

Comment:
there are a couple of areas I would like to clean up and move into a separate method to avoid redundant code / large functions, but I'll make those in a separate PR